documentation/asciidoc/topics/yaml/custom_code_pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: datagrid-libs-pod
+spec:
+  volumes:
+    - name: lib-pv-storage
+      persistentVolumeClaim:
+        claimName: datagrid-libs
+  securityContext:
+    fsGroup: 1000600000
```

Comment:
@oraNod please pay attention that this value and whole `securityContext` make sense for downstream (OCP) only. Dow we need to add notes for it?

Reply:
nice catch @dmvolod I think we should just remove `spec.securityContext` and let downstream users modify the example we provide. This is a temporary solution until the CLI can load artifacts into PVCs. I'm sure users doing this will be saavy enough to modify the pod yaml if a security context is needed.

Reply:
Good, thanks, @oraNod